plugins/kontent-plugin/README.md

```diff
@@ -0,0 +1,309 @@
+<a href="https://www.gatsbyjs.com">
+  <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
+</a>
+
+# Gatsby Starter Kontent Homepage
+
+Create a homepage using Gatsby and Kontent. This starter demonstrates how to use Kontent to build a homepage and can be customized to match your own visual branding.
+
+[View the Demo](https://gatsbykontenthomepage.gatsbyjs.io/)
+
+**Note:**
+This version of the Kontent homepage starter is written in JavaScript. If you want to use Kontent but TypeScript is more your style, there is also a TypeScript version maintained on [GitHub](https://github.com/gatsbyjs/gatsby-starter-kontent-homepage-ts).
+
+## Quick start
+
+To use this starter, you will need a new or existing [Kontent.ai project][] and then you will be asked for your [project id][].
+
+[kontent.ai project]: https://kontent.ai/learn/tutorials/manage-kontent/projects/manage-projects/#a-create-projects
+[project id]: https://kontent.ai/learn/tutorials/develop-apps/get-content/get-content-items/#a-1-find-your-project-id
+
+1. **Create a Gatsby site**
+
+   Use the Gatsby CLI to get started locally:
+
+   ```sh repo
+   npx gatsby new my-homepage https://github.com/gatsbyjs/gatsby-starter-kontent-homepage
+   ```
+
+1. **Create an empty Kontent project**
```

Comment:
The order of instructions could use some tightening up. Or perhaps just under this bullet point we could include the link for how to create a new project?